hash/nes.xml

```diff
@@ -4687,7 +4688,7 @@ license:CC0
 	</software>
 
 	<software name="bloodwar">
-		<description>Bloody Warriors - Shan-Go no Gyakushuu (Jpn)</description>
+		<description>Bloody Warriors - Shan-Go Troop Strikes Back! (Japan)</description>
```

Comment:
Sorry, are the game titles based on the box or the title screen? The in-game title uses the previous title, not the English one on the box.

Reply:
Right, it's from the physical release's various labels. I'm not sure what the preference is for the long description here, but we do have the Japanese name still in the alt_title.

Reply:
Allright! I wasn't sure which was the preferred title to use in this case.

Reply:
I’ve been dealing with this with the e-kara software.  Often the box art, cartridge and in-game title are all different.  Fortunately you can have multiple `alt_title` elements if necessary.

Reply:
I'm open to doing multiple alt_titles by request. I'm just not OCD enough to go gung ho and document every single variant myself. Tonfuu in this PR here, for example, has the title in various combos of kanji/katakana on the labels.

Reply:
I actually didn't know (or remember tbh) we could do multiple alt titles until cuavas mentioned it just now. I think it would be okay to add the old title as an alt before this PR is closed, but don't worry about other titles, those can be done later. :)

Reply:
K, I'll make the goofy English version an alt_title I guess.

Reply:
I meant keep the goofy English as the main and move the old Japanese transliteration to an alt. unless cuavas disagrees.

Reply:
The main description should be a transliteration of whatever the game was most widely known as for Japanese games.

Also, is the current `alt_title` actually correct?  There seem to be plenty of sources that spell it “ウォリアーズ” (rather than “ウォーリアーズ”).

Reply:
Box art seems to agree that it’s “ウォリアーズ”: https://mobajie.com/famikon/1990_bloody-warriors

From a quick look around, it seems to be known as “ブラッディ・ウォリアーズ シャンゴーの逆襲” for the most part.

Reply:
@cuavas Good catch, yeah the kana is wrong compared with the box. I referenced the physical materials in preparing this PR, but I skimmed past that I guess.

Reply:
Nice, that link you gave is wrong also, just wrong in a different way. It's ウァ not ウォ on the artwork.